### PR TITLE
Enable debug mode for sle migration service

### PR DESCRIPTION
**Pull request by yarunachalam**
VR:https://openqa.suse.de/tests/10459533/logfile?filename=serial_terminal.txt
CASEDIR:https://github.com/yarunachalam/os-autoinst-distri-opensuse.git#debug
Change Request works as expected enable verbose and debug mode in config
file
https://openqa.suse.de/tests/10459533/logfile?filename=serial_terminal.txt
verbose_migration: true
debug: true
Job failure is unrelated to this change

### CODE REVIEW

File: tests/publiccloud/migration.pm [hunk 1]
```diff
@@ -44,6 +45,15 @@ sub run {
 
     $instance->ssh_script_run(cmd => "sudo zypper -n in SLES15-Migration suse-migration-sle15-activation", timeout => 300);
 
+    # Include debug mode
+    if ($not_clean_vm) {
+        $instance->ssh_script_run( cmd => 'sudo touch /etc/sle-migration-service.yml' );
```
Comment (asmorodskyi) [inline tests/publiccloud/migration.pm:50]:
```suggestion
        $instance->ssh_script_run(cmd => 'sudo touch /etc/sle-migration-service.yml');
```

File: tests/publiccloud/migration.pm [hunk 2]
```diff
@@ -21,6 +21,7 @@ use utils;
 use publiccloud::utils;
 
 our $target_version = get_required_var('TARGET_VERSION');
+our $not_clean_vm = get_required_var('PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE');
```
Comment (asmorodskyi) [inline tests/publiccloud/migration.pm:24]:
```suggestion
our $not_clean_vm = get_var('PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE');
```
`get_required_var` will fail test if variable is not defined , I don't think this is necessary

Comment (yarunachalam) [reply tests/publiccloud/migration.pm]: Done

Comment (yarunachalam) [reply tests/publiccloud/migration.pm]: done

**Comment by asmorodskyi** [comment]
> VR:https://openqa.suse.de/tests/10357128#settings CASEDIR:https://github.com/yarunachalam/os-autoinst-distri-opensuse.git#debug Change Request works as expected enable verbose and debug mode in config file https://openqa.suse.de/tests/10357128/logfile?filename=serial_terminal.txt verbose_migration: true debug: true Job failure is unrelated to this change

please provide valid verification run, the one which is specified here is not exists anymore

**Comment by yarunachalam** [comment]
> 

Updated with valid verification run

**Pull request merged**